packages/plugin-core/src/components/lookup/TemplateSelector.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  ConfigUtils,
+  DendronError,
+  DLogger,
+  NoteProps,
+} from "@dendronhq/common-all";
+import { ExtensionProvider } from "../../ExtensionProvider";
+import { NoteLookupProviderUtils } from "./NoteLookupProviderUtils";
+import { HistoryEvent } from "@dendronhq/engine-server";
+import { Logger } from "../../logger";
+import * as vscode from "vscode";
+
+export interface TemplateSelector {
```

Comment:
maybe no need for an interface until we have more than one implementation?

Reply:
If we decide to remove the interface, can we also update the filename to `QuickPickTemplateSelector`

Reply:
makes sense. went with a similar pattern with [HierarchySelector](https://github.com/dendronhq/dendron/blob/feat/create-new-with-template/packages/plugin-core/src/components/lookup/HierarchySelector.ts#L10:L10) but the interface is not needed here too